### ISSUE TITLE
LastUsed in service dump has two formats

### ISSUE THREAD

**Issue by Lisimbas** [ubuntu/zsys#152]
**Describe the bug**
The LastUsed field in `zsysctl service dump` is sometimes a UNIX time stamp, and sometimes in ISO 8601 format, depending on where it's used. See for example this snippet:

`
            "user1": {
               "ID": "rpool/USERDATA/user1_abcd",
               "LastUsed": "2018-12-10T13:20:44+01:00",
               "Datasets": {
                  "rpool/USERDATA/user1_abcd": [
                     {
                        "Name": "rpool/USERDATA/user1_abcd",
                        "Mountpoint": "/home/user1",
                        "CanMount": "on",
                        "LastUsed": 1544444444,
                        "BootfsDatasets": "rpool/ROOT/ubuntu_1234"
                     }
                  ]
               }
            },
`

**To Reproduce**
`zsysctl service dump | grep LastUsed | less`

**Expected behavior**
I would expect the format to always be the same. Preferably all UNIX time stamps as those are the easiest to work with.

**Installed versions:**
 - OS: Ubuntu 20.04 LTS
- Zsysd running version: 0.4.5

**Comment by didrocks** [comment]
Good catch! I think we want an option for both format (as dumped can be used to have readable time) for user.

Basically, you have both here:
- the state time is in unix format
- LastUsed of the first dataset of a main route of a state is unix (which is exactly the same time).
